web/src/page_params.ts

```diff
@@ -24,6 +24,7 @@ export const page_params: {
     }[];
     delivery_email: string;
     development_environment: boolean;
+    furthest_read_time: number;
```

Comment:
This may be `null`.

https://github.com/zulip/zulip/blob/0f1fa8e159038a75cd026dbe317f0f4869c63e98/zerver/lib/home.py#L44-L52